index.md

```diff
@@ -35,9 +35,9 @@ deck: "Easy, compliant customer feedback for federal agencies."
       Reinvest in Programs
```

Comment:
I'd consider changing `Reinvest in Programs` to `Start collecting Feedback`

cc @philipashlock

Reply:
Yeah, I'm supportive of that change. I think we should also make that one the first column

Reply:
Updated 3rd column to first column, after updating its Title.